assets/stylesheets/main.css

```diff
@@ -0,0 +1,209 @@
+/*** Project Meta ***/
```

Comment:
- **[OPTIONAL]** It is not required to make your page look good in small screens, but if you want to learn to do that, follow the next tips in inline comments, they will be numbered from 1 to 3
  Your page actually: 
![image](https://user-images.githubusercontent.com/53256139/88986719-67b58500-d299-11ea-9ec1-423e5805e24d.png)